bootstrap-salt.sh

```diff
@@ -2663,6 +2669,11 @@ install_debian_deps() {
         __PIP_PACKAGES="${__PIP_PACKAGES} 'requests>=$_PY_REQUESTS_MIN_VERSION'"
     fi
 
+    if [ "$DISTRO_MAJOR_VERSION" -gt 8 ]; then
```

Comment:
Sorry I was caught up in work. I will remove the PR, test Debian again and put in a new PR in for Ubuntu only. I think that sounds like a plan.